src/components/CustomerQuota/NotEligibleCard.tsx

```diff
@@ -18,10 +18,11 @@ const NotEligibleTransactionTitle: FunctionComponent = () => {
 };
 
 const NotEligibleTransactionDescription: FunctionComponent = () => {
-  const { i18nt } = useTranslate();
+  const { i18nt, c13nt } = useTranslate();
   return (
     <AppText style={{ marginBottom: size(1) }}>
-      {`${i18nt("notEligibleScreen", "logAppeal")}`}
+      {`${c13nt("notEligibleDescription")}` ??
```

Comment:
This doesn't work as expected. Try `${undefined}` or `${null}`, it'll just print "undefined" or "null". The fallback will never be reached.